src/common/version.ts

```diff
@@ -1,7 +1,11 @@
 // This is in its own file so that updates to the version value don't cause a
 // reload of everything.
 
-export const BUILD_VERSION: string = process.env.VERSION!;
+declare global {
+  var SDK_VERSION: string;
+}
```

Comment:
When we generate .d.ts files, will this end up in there? We don't want to expose this to consumers.

Reply:
Our generated `.d.ts` files aren't currently hooked to anything (at least in our internal usage). Point taken though. This would indeed show up in `@inboxsdk/core/` and pollute consuming projects...

Reply:
[5f7c6a6](https://github.com/InboxSDK/InboxSDK/pull/895/commits/5f7c6a6317dd22c455b315f8aeb84f79b1946731)